.env.development

```diff
@@ -3,6 +3,10 @@ VITE_USE_PROXYURL=true
 VITE_PROXYURL_WMS="https://map.geoportail.lu/ogcproxywms"
 VITE_PROXYURL_REMOTE="https://map.geoportail.lu/httpsproxy"
 
+# Paths for symbols
+VITE_SYMBOL_ICONS_URL="https://map.geoportail.lu/mymaps"
+VITE_SYMBOLS_URL="https://map.geoportail.lu/mymaps/symbols"
```

Comment:
Shouldn't we rather use the migration URLs for dev and staging?

Reply:
It would be better I agree but there are currently CORS issues when calling `https://migration.geoportail.lu/mymaps/symbols` in dev, should I change to migration anyway? (it can be bypassed using a chrome plugin but you have to be aware of that. What do you think?

Reply:
I will merge as is for the demo, there is surely something to do with the url from migration/prod for the symbols, especially for the upload symbols functionality.